homework27/src/ba/bitcamp/homework/tasks/Task2.java

```diff
@@ -0,0 +1,164 @@
+package ba.bitcamp.homework.tasks;
+
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+import javax.imageio.ImageIO;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+
+/**
+ * This class extends JFrame and it is used for making negative from readed
+ * picture. Picture is being shown using GUI
+ * 
+ *
+ */
+public class Task2 extends JFrame {
+	// Declaring variables
+	private static final long serialVersionUID = -5118980704703412553L;
+	private static BufferedImage img = null;
+	private JPanel panel;
+	private static LinkedBlockingQueue<Runnable> queue;
+	private static ArrayList<Worker> workers;
+	private static int a = 0;
+	private static int b = 10;
+
+	public static void main(String[] args) {
+		new Task2();
+	}
+
+	/**
+	 * Default constructor that initializes and sets gui components
+	 */
+	public Task2() {
+		importPicture();
+		panel = new MyPanel();
+		add(panel);
+
+		setSize(1920, 1080);
+		setResizable(false);
+		setLocationRelativeTo(null);
+		setTitle("Negativ Task");
+		setDefaultCloseOperation(EXIT_ON_CLOSE);
+		setVisible(true);
+
+		queue = new LinkedBlockingQueue<Runnable>();
+		// Adding new tasks to queue
+		for (int i = 0; i < 150; i++) {
+
+			queue.add(new Task(a, b));
+			a += 10;
+			b += 10;
+
+		}
+
+		workers = new ArrayList<>();
+		// Adding workers
+		for (int i = 0; i < 8; i++) {
+			Worker s = new Worker();
+			s.start();
+			try {
+				s.join();
+			} catch (InterruptedException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
```

Comment:
Exceptions like this should never be just logged, action is required (do something with it, fail the process).